### PR TITLE
Add Gplates package. Plate Tectonics on a sphere.

### PR DESCRIPTION
**Pull request by rbeucher**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rbeucher** [comment]
@conda-forge/cpp-half . 

Hi All, I am putting together a recipe for a package. I got it to work using a debugging environment conda debug -c conda-forge my_recipe but when I try to build it using conda build ... I get this

Traceback (most recent call last):
  File "/home/romain/anaconda3/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/home/romain/anaconda3/lib/python3.8/site-packages/conda_build/cli/main_build.py", line 474, in main
    execute(sys.argv[1:])
  File "/home/romain/anaconda3/lib/python3.8/site-packages/conda_build/cli/main_build.py", line 463, in execute
    outputs = api.build(args.recipe, post=args.post, test_run_post=args.test_run_post,
  File "/home/romain/anaconda3/lib/python3.8/site-packages/conda_build/api.py", line 186, in build
    return build_tree(
  File "/home/romain/anaconda3/lib/python3.8/site-packages/conda_build/build.py", line 3008, in build_tree
    packages_from_this = build(metadata, stats,
  File "/home/romain/anaconda3/lib/python3.8/site-packages/conda_build/build.py", line 2131, in build
    utils.check_call_env(cmd, env=env, rewrite_stdout_env=rewrite_env,
  File "/home/romain/anaconda3/lib/python3.8/site-packages/conda_build/utils.py", line 410, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/home/romain/anaconda3/lib/python3.8/site-packages/conda_build/utils.py", line 390, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/home/romain/anaconda3/conda-bld/gplates_1604372901651/work/conda_build.sh']' returned non-zero exit status 2.

**Review by chrisburr — CHANGES_REQUESTED** [review]
Make sure to rebase this branch to remove `gplates_src_r19627.zip` to avoid inflating the size of the main branch

**Comment by rbeucher** [comment]
Yes but for now I don't have an official link to provide...

File: recipes/gplates/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - svn
+
+  host:
+    - mesa-libgl-devel-cos7-x86_64
+    - libglvnd-glx-cos7-x86_64
```
Comment (chrisburr) [inline recipes/gplates/meta.yaml:29]: See https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl

File: recipes/gplates/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - cmake
+    - make
+    - svn
+
+  host:
+    - mesa-libgl-devel-cos7-x86_64
+    - libglvnd-glx-cos7-x86_64 
+    - glew
+    - python
+    - boost
```
Comment (chrisburr) [inline recipes/gplates/meta.yaml:32]: If you're not using Boost.Python you should use `boost-cpp`

**Comment by rbeucher** [comment]
Thanks for all the suggestions @chrisburr 
I have made some progress. It compiles now but fails at the linking stage...

`/home/romain/anaconda3/conda-bld/gplates_1604458035096/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/9.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libGLdispatch.so.0, needed by /usr/lib/libOpenGL.so, not found (try using -rpath or -rpath-link)`

Any idea where I can get libGLdispatch.so.0 ?

Thanks for your help!

**Comment by chrisburr** [comment]
Based on what's currently here, you should remove `mesa-libgl-devel-cos7-x86_64` and `libglvnd-glx-cos7-x86_64 ` and add the YUM requirements file (next to the `meta.yaml`) that is mentioned in https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl

**Pull request closed without merging**